app/players/player.py

```diff
@@ -0,0 +1,14 @@
+from abc import abstractmethod, ABC
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating() -> None:

```

Comment:
The `get_rating` method should include `self` as its first parameter, like so: `def get_rating(self) -> None:`. This is necessary for instance methods in Python.